src/test/test_helpers.c

```diff
@@ -125,6 +125,22 @@ connection_write_to_buf_mock(const char *string, size_t len,
   buf_add(conn->outbuf, string, len);
 }
 
+char *
+buf_get_contents(buf_t *buf, size_t *sz_out)
+{
```

Comment:
Add `tor_assert(buf)` and `tor_assert(sz_out)` ?

Reply:
✓